src/PolicyInsights/PolicyInsights/help/Get-AzPolicyState.md

```diff
@@ -242,6 +242,29 @@ PS C:\> Get-AzPolicyState -ResourceId "/subscriptions/fff10b27-fff3-fff5-fff8-ff
 
 Gets latest policy state records generated in the last day for the specified resource and expand policyEvaluationDetails.
 
+### Example 23: Get latest component policy states for a resource (eg. vault) and policy assignment referencing a resource provider mode policy definition
```

Comment:
Can we simplify the title to 

Get latest component policy states for a resource (eg. vault) gjven a resource provider mode policy assignment?

Reply:
how do you suggest I word it for initiatives, since an initiative can contain a mix of resource manager and resource provider mode policies?